README.md

```diff
@@ -69,6 +69,16 @@ mock "aws_vpcs" "selected" {
 }
 ```
 
+### Mock return values of resources
+
+Terraform code :
+```
+data "aws_vpcs" "selected" {
+  tags = {
+    service = "secure"
+  }
+}
```

Comment:
This makes me confused, the title says resources but the example shows a single data resource and no mocking